front/app/components/IdeaCard/IdeaCardFooter.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FormEvent } from 'react';
+
+// components
+import CommentCount from './CommentCount';
+
+// types
+import { IIdea } from 'api/ideas/types';
+
+// styles
+import styled from 'styled-components';
+import {
+  Button,
+  colors,
+  useBreakpoint,
+} from '@citizenlab/cl2-component-library';
+import { IOpenPostPageModalEvent } from 'containers/App';
+import eventEmitter from 'utils/eventEmitter';
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+const Footer = styled.footer`
+  display: flex;
+  justify-content: space-between;
+  margin-right: auto;
+  margin-left: auto;
+  margin-top: 16px;
+`;
+
+interface Props {
+  idea: IIdea;
+  showCommentCount: boolean;
+}
+
+const IdeaCardFooter = ({ idea, showCommentCount }: Props) => {
+  const { formatMessage } = useIntl();
+  const isSmallerThanTablet = useBreakpoint('tablet');
+
+  const onReadMoreClick = (event: FormEvent) => {
+    event.preventDefault();
+
+    eventEmitter.emit<IOpenPostPageModalEvent>('cardClick', {
+      id: idea.data.id,
```

Comment:
I thought about just redirecting to the idea page here, but this eventEmitter is used elsewhere as well and allows the user to view the idea (in the modal) and then close it and have the exact same view of the idea list as they had previously. So I decided to use the same approach here as that's a better experience.